modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -545,20 +539,20 @@ class ONNXYoloV3 : public ONNXWithRemap {
 public:
     std::vector<cv::Mat> ins;
 
-private:
-    virtual void SetUp() {
+    void setInput(const cv::Mat& src) {
         const int yolo_in_h = 416;
         const int yolo_in_w = 416;
         cv::Mat yolov3_input, shape, prep_mat;
-        cv::resize(in_mat1, yolov3_input, cv::Size(yolo_in_w, yolo_in_h));
+        cv::resize(src, yolov3_input, cv::Size(yolo_in_w, yolo_in_h));
```

Comment:
Why it's renamed to `src` ?

Reply:
It was `SetUp()` function, will be better that it be a function that creates YoloV3 input based on Mat. This allows avoid problems in the future. Therefore it is `src`-parameter not FIXTURE's `in_mat`.